Rakefile

```diff
@@ -55,11 +55,12 @@ task(:commits) do
   %x{git log --no-merges --pretty=%s #{commit_range}}.each_line do |commit_summary|
     # This regex tests for the currently supported commit summary tokens.
     # The exception tries to explain it in more full.
-    if /^\((maint|packaging|doc|docs|modules-\d+)\)|revert/i.match(commit_summary).nil?
+    if /^\((maint|packaging|doc|docs|(modules|pe)-\d+)\)|revert/i.match(commit_summary).nil?
```

Comment:
I think we decided to allow only `MODULES` for this repos to be easier to track them, 
Upgrade support is also for FOSS but PE tickets are not public.